lib/balena.coffee

```diff
@@ -81,6 +75,7 @@ getSdk = (opts = {}) ->
 		apiVersion: 'v5'
 
 	if opts.isBrowser
+		{ notImplemented } = require('./util')
```

Comment:
This is beneficial because with the new lazy loading it means there's a lot we can actually do without ever triggering utils to load (and it's fairly costly because of memoizee and cloneDeep requires)